ci/jobs/pull_request.yml

```
@@ -0,0 +1,50 @@
+jobs:
+  - name: dataworks-analytical-service-infra-pr
+    plan:
+      - get: dataworks-analytical-service-infra-pr
+        trigger: true
+        version: every
+      - put: dataworks-analytical-service-infra-pr
+        params:
+          context: $BUILD_JOB_NAME
+          path: dataworks-analytical-service-infra-pr
+          status: pending
+      - get: dataworks-analytical-service-infra-client-release
+        version: { tag: '0.0.1' }
+      - .: (( inject meta.plan.terraform-bootstrap ))
+        params:
+          DEPLOY_PATH: infra
+        input_mapping:
+          dataworks-analytical-service-infra: dataworks-analytical-service-infra-pr
+      - task: terraform-plan-infra
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: ((terraform.repository))
+              version: ((terraform12.version))
+              tag: ((terraform12.version))
+          params:
+            AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
+            AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
```

Any truly secret, secret should be contained in `dataworks-secrets` this is now: 

```suggestion
            AWS_ACCESS_KEY_ID: ((dataworks-secrets.ci_aws_access_key_id))
            AWS_SECRET_ACCESS_KEY: ((dataworks-secrets.ci_aws_secret_access_key))
```